pygmt/src/blockm.py

```diff
@@ -70,6 +70,7 @@ def _blockm(block_method, data, x, y, z, outfile, **kwargs):
 @use_alias(
     I="spacing",
     R="region",
+    S="summarize",
```

Comment:
As suggested in https://github.com/GenericMappingTools/pygmt/pull/1601#issuecomment-962215838:
```suggestion
    S="summary",
```